src/Bicep.Core/Parser/Parser.cs

```diff
@@ -29,21 +30,31 @@ public Parser(string text)
 
         public ProgramSyntax Program()
         {
-            var statements = new List<SyntaxBase>();
+            var declarations = new List<SyntaxBase>();
+            
             while (!this.IsAtEnd())
             {
                 var statement = Declaration();
```

Comment:
Does `Declaration` always give a declaration (or skipped tokens), or can it return new line tokens?

Reply:
It can contain NewLine Tokens too. Let me rename the variable.

Reply:
Renamed variable and added comments